pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Schemas.java

```diff
@@ -98,6 +98,22 @@
      */
     CompletableFuture<Void> deleteSchemaAsync(String topic);
 
+    /**
+     * Delete the schema associated with a given <tt>topic</tt>.
+     *
+     * @param topic topic name, in fully qualified format
+     * @param force force to delete schema
```

Comment:
```suggestion
     * @param force to delete schema
```

And it's better to provide more information about what is the difference after use force==true, it looks like if force==false, which means it's a mark deletion operation, will not remove any resources, but with force==true, it will remove all the resources used by schema.

Reply:
ok, will add some descriptions